### PR TITLE
resources and one peasant only from command line 

### PR DESCRIPTION
**Pull request by mimooh**
hopefully I correctly used git's magic to work with branches...

### CODE REVIEW

File: scripts/menus/network.lua [hunk 1]
```diff
@@ -512,187 +516,205 @@ function RunServerMultiGameMenu(map, description, numplayers, options)
...
+        revealMapCb(revealmap)
+        options.revealmap = -1
+      elseif (optAutostartNum) then
+        if (optAutostartNum <= readyplayers) then
+          if (startIn < 0) then
+            startIn = 100
+          else
+            startIn = startIn - 1
+            if (startIn == 0) then
+              startFunc() end
```
Comment (timfel) [inline scripts/menus/network.lua]: This doesn't look right ... Shouldn't it be
```lua
if (startIn < 0) then
  startIn = 100
else
  startIn = startIn  - 1
  if (startIn == 0) then
    startFunc()
  end
end
```
?

**Comment by timfel** [comment]
Besides this minor comment, this looks good!

File: scripts/menus/network.lua (hunk 1 above)
Comment (mimooh) [inline scripts/menus/network.lua]: You mean 'startFunc() end' should be split in two lines, right? I agree
with you. It's probably that I was trying to manually indent the code
and saw these 'func() end' elsewere in lua code. Then I used some
external formatter.

Besides the formatting the code works for me. Don't you think these long
functions should be split into more but shorter functions? I could try
to make that.

-- 
Karol Kreński

**Comment by timfel** [comment]
Thanks!

**Pull request merged**